examples/v1alpha2/tls-basic.yaml

```diff
@@ -0,0 +1,25 @@
+kind: Gateway
+apiVersion: gateway.networking.k8s.io/v1alpha2
+metadata:
+  name: tls-basic
+spec:
+  gatewayClassName: acme-lb
+  listeners:
+  - name: foo-https
+    protocol: HTTPS
+    port: 443
+    hostname: foo.example.com
+    tls:
+      certificateRef:
+        kind: Secret
+        group: ""
```

Comment:
consider omitting `group` since the default is "core"

Reply:
I know we once had a default here, but it looks like we've removed it in favor of a generic ObjectRef, so I think this still needs to be specified.